rust/osm2lanes/src/overpass.rs

```diff
@@ -67,7 +74,7 @@ pub async fn get_tags(id: ElementId) -> Result<Tags, reqwest::Error> {
 ///
 /// Unexpected data from overpass and/or openstreetmap.
 ///
-pub async fn get_way(id: ElementId) -> Result<(Tags, Locale), reqwest::Error> {
+pub async fn get_way(id: &ElementId) -> Result<(Tags, Locale), reqwest::Error> {
```

Comment:
this may want to return geometry as well in the future, in order to display it on the slippy map.